cmake/Hunter/config.cmake

```diff
@@ -1 +1,2 @@
-hunter_config(CURL VERSION ${HUNTER_CURL_VERSION} CMAKE_ARGS HTTP_ONLY=ON CMAKE_USE_OPENSSL=OFF CMAKE_USE_LIBSSH2=OFF)
\ No newline at end of file
+hunter_config(CURL VERSION ${HUNTER_CURL_VERSION} CMAKE_ARGS HTTP_ONLY=ON CMAKE_USE_OPENSSL=OFF CMAKE_USE_LIBSSH2=OFF)
+hunter_config(libjson-rpc-cpp VERSION ${HUNTER_libjson-rpc-cpp_VERSION} CMAKE_ARGS TCP_SOCKET_SERVER=ON)
```

Comment:
So TCP socket only, not HTTP?

Reply:
Since the current implementation is claymore compatible, which only supports json-rpc via tcp not http, yes. (ok claymore also works if you connect to the port via http, but thats not json-rpc. it returns the console output via http)

Also HTTP needs more hunter dependencies (microhttpd)
My plan is, once microhttpd is supported to allow different api protocols.
For example 'Prometheus Endpoint')